index.html

```diff
@@ -79,11 +84,48 @@ <h2 class="tonic">Tonic</h2>
                         <li>javascript</li>
                     </ul>
                     <button class="see-project">See Project</button>
+                  </div>
+        </section> 
+        <section class="sec2 desktop">
+          <div class="sec-content">
+            <div class="image-div"> 
+              <img src="images/snapshoot Portfolio.png" alt="image" class="workplace-image">
```

Comment:
- [ ] Kindly consider using the correct path for your image.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201397399-e153cc8c-f1a2-44da-82cc-d7301c3b116e.png)